openmls/src/framing/message.rs

```diff
@@ -224,7 +227,10 @@ impl From<PrivateMessage> for MlsMessageOut {
         let body = MlsMessageBody::PrivateMessage(ciphertext);
 
         Self {
-            mls_message: MlsMessage { body },
+            mls_message: MlsMessage {
+                body,
+                protocol_version: ProtocolVersion::Mls10,
```

Comment:
Here we don't have the version in the message. But we should be able to pass it in without too much churn. I just want to avoid hard coding the version in too many places when we can easily get to it.